.docker/Dockerfile-CPU-Base

```diff
@@ -3,38 +3,144 @@
 # ------------------------------------------------------------------
 # Ubuntu           18.04
 # OpenMPI          latest       (apt)
-# cmake            3.10         (git)
-# MKL              2018.4-057   (apt)
-# arrayfire        3.7.1        (git, CPU backend)
-# oneDNN           2.0          (git)
-# Gloo             b7e0906      (git)
-# libsndfile       4bdd741      (git)
+# cmake            3.10         (oficial binaries)
+# MKL              2020.4-912   (apt)
+# arrayfire        3.7.3        (git, CPU backend)
+# libsndfile       latest       (apt, v1.0.28-4)
+# oneDNN           master       (git)
```

Comment:
let's freeze at `v2.0`, which is what we're using in CI

Reply:
done in https://github.com/facebookresearch/flashlight/pull/401